client/app/scripts/superdesk-packaging/views/sd-package-item-preview.html

```diff
@@ -17,7 +17,7 @@
             <div ng-hide=":: item.headline || item.slugline || item.description" translate>Blank headline received</div>
         </div>
 
-        <a ng-href="#/{{ :: type !== 'composite' ? 'authoring' : 'packaging'}}/{{ :: item.guid }}/view?_id={{ :: item.guid }}" 
+        <a ng-href="#/{{ :: type !== 'composite' ? 'authoring' : 'packaging'}}/{{ :: type !== 'video' || 'picture' ? item.residRef : item.guid}}/view?_id={{ :: item.guid }}" 
```

Comment:
It works in this way if it is picture or video it results item.residRef and if it is neither it wil result item.guid. In my test example, where I have picture, video and text it works in this way.